src/Microsoft.Azure.WebJobs.Extensions.Storage/Blobs/Listeners/BlobListenerFactory.cs

```diff
@@ -32,7 +32,7 @@ internal class BlobListenerFactory : IListenerFactory
         private readonly CloudBlobContainer _container;
         private readonly IBlobPathSource _input;
         private readonly ITriggeredFunctionExecutor _executor;
-        private readonly IHostSingletonManager _singletonManager;
+        private readonly ISingletonManager _singletonManager;
```

Comment:
When this interface was introduced, the "Host" prefix was arbitrarily added - not sure why.